common/src/main/java/tachyon/LeaderInquireClient.java

```diff
@@ -83,6 +84,7 @@ public synchronized String getMasterAddress() {
                 leader = master;
               }
             }
+            LOG.info("The leader master: " + leader);
```

Comment:
Hrmmm, is it too verbose? On line 70, it already logs the list of masters. Unfortunately, that list is not ordered by time, so there is no way to tell which one is the leader. Would this log message be too verbose?